src/main/java/Routes/Router.java

```diff
@@ -55,6 +55,9 @@ private void createRoutes() {
         routes.add(new Route("/these", new TheseController()));
         routes.add(new Route("/requests", new RequestsController()));
         routes.add(new Route("/redirect", new RedirectController()));
+        routes.add(new Route("/file1", new FileController(new FileReader())));
+        routes.add(new Route("/file2", new FileController(new FileReader())));
+        routes.add(new Route("/text-file.txt", new FileController(new FileReader())));
```

Comment:
Be careful not to tie your implementation to the CobSpec tests. The server should be able to serve requests for any file and any directory.